client/servers/create_server_responses.go

```diff
@@ -399,17 +399,17 @@ type CreateServerParamsBodyDataAttributes struct {
 
 	// RAID mode for the server
 	// Enum: [raid-0 raid-1]
-	Raid *string `json:"raid,omitempty"`
+	Raid string `json:"raid,omitempty"`
 
 	// The site to deploy the server
 	// Enum: [ASH BGT BUE CHI DAL FRA LAX LON MEX MEX2 MIA MIA2 NYC SAN SAN2 SAO SAO2 SYD TYO TYO2]
 	Site string `json:"site,omitempty"`
 
 	// SSH Keys to set on the server
-	SSHKeys []string `json:"ssh_keys,omitempty"`
+	SSHKeys []int64 `json:"ssh_keys,omitempty"`
```

Comment:
I assumed this issue was on the API side. The documentation says this attribute should be a string (and in fact it should), but when sending strings to the API, it returns an error saying it expects an array of integers.

I'm going to do some additional tests to confirm if the problem is in fact on the API, if so, we'll need to send integers for now to avoid this issue and consider updating the API.

Reply:
@ynhummel just confirming that the problem is actually on the API. When trying to deploy a server with SSH Keys as an array of strings, it returns the following response:

![image](https://github.com/latitudesh/lsh/assets/6020481/f09b6d28-e487-42df-8c2e-27acbbff483a)

It's curious that this issue hasn't been reported yet, but I'll let the team know about that. In the meantime, you can pass an array of integers to test the servers creation.